portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,11 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function getList() {
```

Comment:
Please add some documentation to this method and maybe rename it to something more meaningful. `getList` is too generic and since the method is not documented it's pretty hard to figure out what it's expected to do.